business/src/entities/Document.js

```diff
@@ -0,0 +1,68 @@
+const joi = require('joi-browser');
+const uuidv4 = require('uuid/v4');
+
+const uuidVersions = {
+  version: ['uuidv4'],
+};
+
+const documentTypes = [
+  'Petition',
+  'Request for Place of Trial',
+  'Statement of Taxpayer Identification Number',
+];
+
+const documentSchema = joi.object().keys({
+  documentType: joi
+    .string()
+    .valid(documentTypes)
+    .required(),
+  documentId: joi
+    .string()
+    .uuid(uuidVersions)
+    .required(),
+  userId: joi
+    .string()
+    // .uuid(uuidVersions)
```

Comment:
Either UUID or some auto incrementing id.  It will probably depend on what aws cognito generates.  Right now a lot of our code is using 'taxpayer' or 'petitionsclerk' for that userId, etc, so we have that commented out just for now to speed up our development.